backend/app/usecase/search/order/created_time.go

```diff
@@ -0,0 +1,43 @@
+package order
+
+import (
+	"sort"
+
+	"github.com/short-d/short/backend/app/entity"
+)
+
+var _ Order = (*CreatedTime)(nil)
+
+// CreatedTime arranges searchable resources based on their created time.
+type CreatedTime struct {
+}
+
+// ArrangeShortLinks arranges shortLinks based on CreatedAt.
+// The returned short links satisfies the created before comparision if CreatedAt
+// is not nil, otherwise it sits on the end of the slices if CreatedAt is nil.
```

Comment:
```suggestion
```
The code seems to be very clear on the behavior of this function. Comments are usually used to explain `why` instead of `what` because everyone who reads the code can understand what but not why.

Reply:
Thank you for the explanation!